service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusServiceTest.java

```diff
@@ -63,6 +66,58 @@ public void emptyDatabase() {
         ehrExtractStatusRepository.deleteAll();
     }
 
+    @Test
+    public void When_FetchDocumentObjectNameAndSize_With_OneMissingAttachment_Expect_Returned() {
+        var inProgressConversationId = generateRandomUppercaseUUID();
+
+        addInProgressTransfer(
+            inProgressConversationId, List.of(
+                EhrExtractStatus.GpcDocument.builder()
+                    .fileName("AbsentAttachment4E0C8345-A9AB-48EA-8882-DC9E9F3F5F60.rtx")
+                    .documentId("4E0C8345-A9AB-48EA-8882-DC9E9F3F5F60")
+                    .contentLength(DEFAULT_CONTENT_LENGTH)
+                    .gpConnectErrorMessage("404 Not Found")
+                    .contentType("application/msword")
```

Comment:
Minor but I think "application/msword" can be defined as a constant as it is used 3 times in the tests.